deployment/stop-monitoring.yaml

```
@@ -0,0 +1,14 @@
+# Stops the health monitoring service.
+#
+# Applies to all hosts by default, unless other nodes are specified using --extra-vars "nodes=..."
+
+---
+- name: Stop Health monitoring
+  hosts: "{{nodes | default('all')}}"
+  gather_facts: False
+  become: False
+  tasks:
+
+    - name: Stop Monitoring server
+      shell: "pkill spacenet-health"
```

It might be better to add a timeout after which the process is killed `-9`. Just in case. See [here](https://github.com/consensus-shipyard/spacenet/blob/main/deployment/scripts/kill.sh). No need to create another script file though. Probably can be done as follows (but pls check if it actually works before using it).

```suggestion
      shell: "killall spacenet-health; sleep 3; killall -9 spacenet-health; true"
```